civicrm.php

```diff
@@ -737,17 +737,11 @@ public function enable_translation() {
+
```

Comment:
👍 for SVG.

A slight nitpick... this does add 18kb of code in two source files, and they're duplicates. Would make sense to do something like:

```php
$civilog = file_get_contents(__DIR__ . "civilog.svg.b64");
```

or maybe

```php
$civilog = 'data:image/svg+xml;base64,' . base64_encode(file_get_contents(__DIR__ . "civilog.svg"));
```

Reply:
I was thinking about the second, but I didn't want to require the server to encode the file every time the admin menu appears.  I never really thought about splitting the difference by going about it the first way, but I think I'll do that.